include/evmc/evmc.hpp

```diff
@@ -328,7 +328,7 @@ constexpr auto make_result = evmc_make_result;
 ///
 /// This is a RAII wrapper for evmc_result and objects of this type
 /// automatically release attached resources.
-class result : private evmc_result
+class Result : private evmc_result
```

Comment:
What is the reasoning behind this? Complex data types are upper case, simple data types (bytes32 for example) are lowercase?

Reply:
For consistency with `evmc::VM`, `evmc::Host`, `evmc::HostInterface`, etc. Not that we have perfect consistency but upper case names are easier to use in C++. @yperbasis was in favor.